### ISSUE TITLE
A11y - Onramp - Restart the guide links should match visually

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#118550]
## Issue

The "Restart the guide" link under "your available options" should have the same styling as the "Want to explore other pathways" Restart the guide link. 

If a user sees two links that look different, they might reasonably assume they do different things, even if the text is the same. This is particularly important for:
- Users with cognitive disabilities who rely on consistent patterns
- Users navigating via visual cues, not just text
- People who use assistive technologies alongside visual scanning

Accessibility Defect Severity: [4: Low. Consider fixing or exploring in 2-4 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-4)
WCAG Success Criteria: [WCAG 3.2.4 Consistent Identification - Level AA](https://www.w3.org/TR/WCAG22/#consistent-identification)

<img width="1440" height="842" alt="Image" src="https://github.com/user-attachments/assets/1c62c224-5110-41ac-ad44-6622ff8ceaac" />

<img width="1560" height="458" alt="Image" src="https://github.com/user-attachments/assets/78c0ba37-cc7f-4467-884a-751f4fb4f9f3" />

**Comment by conchette** [comment]
@randimays and @kbermangestring I don't see any issues with styling it the same as the  "Want to explore other pathways" Restart the guide link.

**Comment by randimays** [comment]
@conchette Per the DS guidance, there should only be one green action link per page ([docs](https://design.va.gov/components/link/action#usage)):

> Primary calls to action. A Link - Action - Primary (green) can be used for the primary call to action on a page. A Primary entry variation is preferred, but the Primary variation is allowed. Use only one Link - Action - Primary per page.

I realize we are already breaking this convention in a couple of places, so maybe that's ok?

**Comment by kbermangestring** [comment]
@conchette i think we can change the restart the guide to a secondary action link. This would resolve the multiple action links on the same page for all non-DR summary screens. we would still have multiple (within the cards) on the DR dynamic master summary screens, but that is something we could discuss further. Thoughts?

**Comment by bacitracin** [comment]
Since dupe link has been removed from the designs, closing this ticket
